styles/globals.css

```diff
@@ -19,12 +19,12 @@ a {
   box-sizing: border-box;
 }
 
-/* @media (prefers-color-scheme: dark) {
```

Comment:
We aren't supporting dark mode as of now, please keep this media-query commented